recipes/openmesh/debug-bindings.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/src/Python/CMakeLists.txt b/src/Python/CMakeLists.txt
+index e622ab8..3985311 100644
+--- src/Python/CMakeLists.txt
++++ src/Python/CMakeLists.txt
+@@ -36,7 +36,7 @@ IF(OPENMESH_BUILD_PYTHON_BINDINGS)
+ 		IF(Boost_FOUND)
+ 			MESSAGE(STATUS "Looking for Boost Python -- found")
+ 
+-			MESSAGE(STATUS "Checking the Boost Python configuration")
++			MESSAGE(STATUS "Checking the Boost Python configuration Boost_LIBRARIES=${Boost_LIBRARIES}")
```

Comment:
Are these messages really necessary? Would prefer not to maintain a patch if possible.